openapi/kas-fleet-manager.yaml

```diff
@@ -1019,7 +1019,87 @@ paths:
       security:
         - Bearer: [ ]
       tags:
-        - enterprise-dataplane-clusters  
+        - enterprise-dataplane-clusters
+  /api/kafkas_mgmt/v1/clusters/{id}:
+    delete:
+      tags:
+        - enterprise-dataplane-clusters
+      operationId: deleteEnterpriseClusterById
+      parameters:
+        - in: query
+          name: async
+          description: Perform the action in an asynchronous manner
+          schema:
+            type: boolean
+          required: true
+        - in: query
+          name: force
+          description: |-
+            When provided with value: true - enterprise cluster will be deleted alongside all kafkas present on the cluster.
+            When skipped and enterprise cluster has any kafkas associated with it, the request will fail.
+          schema:
+            type: boolean
+          required: false
```

Comment:
@pb82 @pawelpaszki 

I understand the motivation behind the `force` flag but I am wondering if the alternative of not deleting the data plane cluster if it contains kafka was considered and only letting the user explicitly delete the Kafka.

Reply:
I'll let Peter answer. I am fine either way. I see this endpoint being used mostly via Ui. if thats the case, there should be a warning stating that `force:true` will remove all kafkas

Reply:
Thanks @pawelpaszki let's see what @pb82 says. Perhaps this was discussed already in some other channels and I could missing some context.